addons/hitreactions/XEH_postInit.sqf

```diff
@@ -0,0 +1,50 @@
+#include "script_component.hpp"
+
+if (GVAR(weaponDropChanceGunHit) + GVAR(weaponDropChanceArmHit) == 0) exitWith {};
+
+["ace_firedNonPlayer", {
+    params ["", "", "", "", "", "", "_projectile"];
+    private _roll = random 1;
+    if (!(_roll < GVAR(weaponDropChanceGunHit) || _roll < GVAR(weaponDropChanceArmHit))) exitWith {};
```

Comment:
There is no point in having 2 separate settings when you combine them.

Either you need to check the probabilities in `FUNC(checkWeaponDrop)` or you have one setting.

I tested this by setting `Weapon Drop Chance (Gun Hit)` to 100 % and the other to 0%. When a unit is shot in the arm, he drops the weapon.

Reply:
Resolved in latest commit, think there may be a faster way to do it though